pkg/reconciler/autoscaling/kpa/kpa.go

```diff
@@ -123,22 +123,26 @@ func (c *Reconciler) ReconcileKind(ctx context.Context, pa *autoscalingv1alpha1.
 		return fmt.Errorf("error scaling target: %w", err)
 	}
 
-	mode := nv1alpha1.SKSOperationModeServe
+	mode := nv1alpha1.SKSOperationModeProxy
+
 	switch {
 	// When activator CA is enabled, force activator always in path.
 	// TODO: This is a temporary state and to be fixed.
 	// See also issues/11906 and issues/12797.
 	case len(config.FromContext(ctx).Network.ActivatorCA) > 0:
 		mode = nv1alpha1.SKSOperationModeProxy
-	// We put activator in the serving path in the following cases:
-	// 1. The revision is scaled to 0:
-	//   a. want == 0
-	//   b. want == -1 && PA is inactive (Autoscaler has no previous knowledge of
-	//			this revision, e.g. after a restart) but PA status is inactive (it was
-	//			already scaled to 0).
-	// 2. The excess burst capacity is negative.
```

Comment:
nit: I wonder if we should keep this comment block (maybe move it to just before the `switch` statement)? It's a nice statement on when we want the activator in the path, (though I guess since we're defaulting to Proxy mode, it's maybe not the right want to frame it any more...

Reply:
yeah I split this out into the three comments for the three cases - let me know if you think they need more clarity

Reply:
Nope, no issues there, I think they're very clear.